CMakeLists.txt

```diff
@@ -127,6 +127,15 @@ IF(NOT WIN32)
   ELSE()
     MESSAGE(FATAL_ERROR "Driver Manager was not found")
   ENDIF()
+
+  # Looking for iconv files
+  INCLUDE(${CMAKE_SOURCE_DIR}/cmake/FindIconv.cmake)
+  IF(ICONV_FOUND)
+    INCLUDE_DIRECTORIES(${ICONV_INCLUDE_DIR})
+    SET(PLATFORM_DEPENDENCIES ${PLATFORM_DEPENDENCIES} ${ICONV_LIBRARIES})
```

Comment:
No problem.. I've changed that block from IF(NOT WIN32) to a separate IF(APPLE) block.